README.rst

```diff
@@ -12,6 +12,12 @@ Driver Mantras
 When developing specifications -- and the drivers themselves -- we follow the
 following principles:
 
+
+Be Idiomatic
+~~~~~~~~
```

Comment:
FYI: RST heading modifiers should extend to the length of their text to avoid a warning.